ccc/github.py

```diff
@@ -55,6 +57,8 @@ def github_api_ctor(
     In case github_url does not refer to github.com, the c'tor for GithubEnterprise is
     returned with the url argument preset, thus disburdening users to differentiate
     between github.com and non-github.com cases.
+
+    If es_client is provided, session requests are logged to ES.
```

Comment:
it is okay to abbreviate elastisearch-client to es_client, esp. considering there is a typehint. However, please do not abreviate Elasticsearch to just ES in the docstr.

some more nits:
- this is a plain function; es_client is a formal parameter of this function. I think it is more conventional to speak of "passing an argument", or "giving" (as in: if es_client is passed / given / non-none)
- while it is technically correct that "session requests" will be logged, this is not the relevant semantics, but an implementation detail. what happens is an elasticsearch-client is passed, is any api-requests issues by github-api-clients created by the returned function will be logged (to elasticsearch). we might shorten this to just "all requests against github-api will be logged", for example (document WHAT/WHY, not HOW - the latter is already written down in the actual code; documentation should almost never repeat the code)